.github/workflows/docs_build_and_deploy.yml

```diff
@@ -15,7 +15,15 @@ on:
   workflow_dispatch:
 
 jobs:
+  linting:
+    # scheduled workflows should not run on forks
+    if: (${{ github.event_name == 'schedule' }} && ${{ github.repository_owner == 'neuroinformatics-unit' }} && ${{ github.ref == 'refs/heads/main' }}) || (${{ github.event_name != 'schedule' }})
+    runs-on: ubuntu-latest
+    steps:
+      - uses: neuroinformatics-unit/actions/lint@v1
```

Comment:
> Why are we using v1 of the linting action instead of v2?

I blindly copied and pasted from somewhere else!

> Also what does the linting accomplish on this repo, since there is barely any Python source code?

It's kinda overkill, but it seemed like a good idea to run pre-commit in a standardised way.

I've updated the version now